### DISCUSSION TITLE
Where is the Enkoder-window?

### DISCUSSION THREAD

**Discussion by sofdwere** [NickeManarin/ScreenToGif#1234] [Q&A] [answered]
or is this no longer available in the software?

![8c67a8d4-8724-11e6-9c4d-cec566b4a494](https://github.com/NickeManarin/ScreenToGif/assets/107915404/206e4f7d-c35d-4631-a280-d83f50c3bfd9)

**Comment by NickeManarin** [comment, marked as answer]
It's still available, it's just not enabled by default.

![image](https://github.com/NickeManarin/ScreenToGif/assets/14798947/e1b2294b-804b-43f0-abd5-54d9217c8827)
